stable/kube2iam/Chart.yaml

```diff
@@ -1,5 +1,6 @@
+apiVersion: v1
 name: kube2iam
-version: 0.11.0
+version: 1.0.0
```

Comment:
Shouldn't this be `0.11.1`?

Reply:
we agreed some time ago that all charts in the stable folder should be at least 1.0.0 @mattfarina